packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -107,6 +109,39 @@ class PurchaseParam {
 
   /// The 'sandboxTesting' is only available on iOS, set it to `true` for testing in AppStore's sandbox environment. The default value is `false`.
   final bool sandboxTesting;
+
+  /// The 'changeSubscriptionParam' is only available on Android, for upgrading or
+  /// downgrading an existing subscription. This does not require on iOS since
+  /// Apple provides a way to group related subscriptions together in iTunesConnect.
+  /// So when a subscription upgrade or downgrade is requested, Apple finds the old
+  /// subscription details from the group and handle it automatically.
+  final ChangeSubscriptionParam changeSubscriptionParam;
+}
+
+/// The parameter object for upgrading or downgrading an existing subscription so that
```

Comment:
nits: dart doc should start with a single sentence summary and an empty line between summary and description.

Reply:
Done